experimental/dds/tree2/src/test/schemaBuilder.spec.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+import { ValueSchema } from "../core";
+import { SchemaBuilder } from "../feature-libraries";
+import { brand } from "../util";
+
+describe("SchemaBuilder unit tests", () => {
+	describe("intoDocumentSchema", () => {
+		it("Simple", () => {
+			const schemaBuilder = new SchemaBuilder("test");
+			const leafSchema = schemaBuilder.leaf("leaf", ValueSchema.Boolean);
+			const schema = schemaBuilder.intoDocumentSchema(
```

Comment:
The simple test case for `intoLibrary()` is probably almost identical to this test if you feel like adding it now.